CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+### Behavior Changes
+* `LogBatch::put` and `LogBatch::put_message` allow to put key start with `__` but not equals the `__{ATOMIC_GROUP_KEY}`(0x5f 0x5f 0x01).
```

Comment:
I think this is too verbose, we have never described the exact internal prefix before.

How about "Reduce the scope of keys reserved for internal use."